src/queries/index.ts

```diff
@@ -0,0 +1,21 @@
+import * as user from './user-queries';
+import * as action from './action-queries';
+import * as actionType from './action-type-queries';
+import * as feedItem from './feed-item-queries';
+import * as comment from './comment-queries';
```

Comment:
Think these could be simple re-exports.

Reply:
The imports are still needed for the `interface Queries`, no? Idk if you can somehow automatically build that definition.

Reply:
You had a fix in your PR. Apparently changing the import style makes `typeof` work. I'll copy that to this PR.